im.pidgin.Pidgin3.yml

```diff
@@ -0,0 +1,128 @@
+app-id: im.pidgin.Pidgin3
+runtime: org.gnome.Platform
+runtime-version: '47'
+sdk: org.gnome.Sdk
+command: pidgin3
+finish-args:
+  # X11 + XShm access
+  - "--share=ipc"
+  - "--socket=fallback-x11"
+  # Wayland access
+  - "--socket=wayland"
+  # GPU acceleration if needed
+  - "--device=dri"
+  # Needs to talk to the network:
+  - "--share=network"
+  # Audio
+  - "--socket=pulseaudio"
+  # Notifications (purple-toast)
+  - "--talk-name=org.freedesktop.Notifications"
+  # Accessibility
+  - "--talk-name=org.a11y.Bus"
+  # libsecret plugin
+  - "--talk-name=org.freedesktop.secrets"
+  # kwallet plugin
+  - "--talk-name=org.kde.kwalletd5"
+  - "--talk-name=org.kde.kwalletd6"
+  # Bonjour
+  - "--system-talk-name=org.freedesktop.Avahi"
+  # Plugin Paths
+  - "--env=PURPLE_PLUGIN_PATH=/app/plugins/purple-3"
+  - "--env=PIDGIN_PLUGIN_PATH=/app/plugins/pidgin-3"
+
+add-extensions:
+  im.pidgin.Purple3.Plugin:
+    directory: plugins
+    version: default
+    add-ld-path: lib
+    merge-dirs: purple-3
```

Comment:
supports multiple values, so combine the two https://docs.flatpak.org/en/latest/extension.html#extension-point

Reply:
I'm not sure what you mean here, but let me add some back story.

Libpurple is a library that is currently in the pidgin repository. There are other frontends to libpurple that can and will be eventually released (just like purple2, see finch). So the idea here was to have a shared extension point for libpurple only plugins as at some point in the future it will be pulled out to it's own repository as well.

Reply:
> So the idea here was to have a shared extension point for libpurple only plugins as at some point in the future it will be pulled out to it's own repository as well.

what do you mean by "libpurple only plugins"? Are they based on libpurple library? 

But if they are both going to be used by Pidgin, then why would you need 2 separate extension points?

Reply:
libpurple plugins work with any application that uses libpurple 3. Right now there is only one, Pidgin 3, but we do have an alternative user interface in the works that would be using those plugins as well but not the plugins that are for Pidgin 3 as the application is different and they have different api/abis.

Reply:
Ok then it's fine.

Reply:
Maybe describing how this works with purple2 will help.

In debian we can see that finch and pidgin both depend on libpurple, but finch doesn't include gtk. So any pidgin plugin that pulls in gtk won't work in finch and vice versa. But any plugin written that only uses libpurple api/abi will work in both which is why it has a separate plugin directory from the user interfaces.

```
$ apt-cache show finch | grep -i Depends
Depends: pidgin-data (<< 2.14.13-z), pidgin-data (>= 2.14.13), libc6 (>= 2.34), libglib2.0-0t64 (>= 2.76.0), libgnt0t64 (>= 2.14.0), libgstreamer1.0-0 (>= 1.0.0), libncursesw6 (>= 6), libpurple0t64 (>= 2.8.0), libtinfo6 (>= 6)
$ apt-cache show pidgin | grep -i Depends
Depends: pidgin-data (<< 2.14.13-z), pidgin-data (>= 2.14.13), perl-base (>= 5.40.0-6), perlapi-5.40.0, libatk1.0-0t64 (>= 1.12.4), libc6 (>= 2.34), libcairo2 (>= 1.2.4), libdbus-1-3 (>= 1.9.14), libgdk-pixbuf-2.0-0 (>= 2.22.0), libglib2.0-0t64 (>= 2.76.0), libgstreamer-plugins-base1.0-0 (>= 1.0.0), libgstreamer1.0-0 (>= 1.0.0), libgtk2.0-0t64 (>= 2.24.0), libgtkspell0 (>= 2.0.10), libice6 (>= 1:1.0.0), libpango-1.0-0 (>= 1.18.0), libpurple0t64 (>= 2.14.0), libsm6, libx11-6, libxss1
```